### PR TITLE
Allow setting of xhr timeout for all request

### PR DESCRIPTION
**Pull request by escalona**
In response to #58

### CODE REVIEW

**Comment by escalona** [comment]
If this gets accepted I'll create a PR for fluxible-plugin-fetchr to allow setting this option

File: libs/fetcher.client.js [hunk 1]
```diff
@@ -146,6 +147,7 @@ Queue.prototype = {
      * @class FetcherClient
      * @param {object} options configuration options for Fetcher
      * @param {string} [options.xhrPath="/api"] The path for XHR requests
+     * @param {number} [options.xhrTimout=3000] Timeout for all XHR requests
```
Comment (Vijar) [inline libs/fetcher.client.js]: Perhaps `Timeout in milliseconds...` to specify the units

**Comment by Vijar** [comment]
@escalona 
We should also get rid of `DEFAULT_CONFIG.timeout`([here](https://github.com/yahoo/fetchr/blob/master/libs/util/http.client.js#L93) and [here](https://github.com/yahoo/fetchr/blob/master/libs/util/http.client.js#L23)) now that we always fallback to the `xhrTimeout` value.

**Comment by escalona** [comment]
@Vijar Good catch. Updated http.client

**Comment by Vijar** [comment]
:ship:

**Pull request merged**